include/glk/texture.hpp

```diff
@@ -19,6 +19,7 @@ class Texture {
   GLuint id() const;
   Eigen::Vector2i size() const;
   void set_size(const Eigen::Vector2i& size);
+  void set_color(const void* pixels);
```

Comment:
I would like to avoid adding `set_color()` to `Texture` for now because re-using the allocated texture is often prone to be buggy. We may consider adding this feature later with buffer size check for safety.

Reply:
> I would like to avoid adding set_color() to Texture for now because re-using an allocated texture is often prone to be buggy. 

I have never used Open GL so Thank you for your technical adovice.
I agree to your sugestion and I remove this function.